cli/cmd/generate_azure.go

```diff
@@ -0,0 +1,680 @@
+package cmd
+
+import (
+	"fmt"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/imdario/mergo"
+	"github.com/spf13/cobra"
+
+	"github.com/lacework/go-sdk/lwgenerate/azure"
+	"github.com/pkg/errors"
+)
+
+var (
+	// Define question text here so they can be reused in testing
+	QuestionAzureEnableConfig = "Enable Azure Config Integration?"
+	QuestionAzureConfigName   = "Specify Name of the Config"
+	QuestionEnableActivityLog = "Enable Azure Activity Log Integration?"
+	QuestionActivityLogName   = "Specify name of the Activity Log"
+
+	QuestionAzureAnotherAdvancedOpt      = "Configure another advanced integration option"
+	QuestionAzureConfigAdvanced          = "Configure advanced integration options?"
+	QuestionAzureCustomizeOutputLocation = "Provide the location for the output to be written:"
+
+	// Active Directory
+	QuestionEnableAdIntegration = "Create Active Directory Integration?"
+	QuestionADApplicationPass   = "Specify the password of an existing Active Directory application"
+	QuestionADApplicationId     = "Specify the ID of an existing Active Directory application"
+	QuestionADServicePrincpleId = "Specify the Service Principle ID of an existing Active Directory application"
+
+	// Storage Account
+	QuestionUseExistingStorageAccount   = "Enable use of existing Storage Account?"
+	QuestionAzureRegion                 = "Specify the Azure region to be used by Storage Account logging"
```

Comment:
```suggestion
	QuestionAzureRegion                 = "Specify the azure region in which all resources should be created"
```

Reply:
I think this does relate to the storage logs are, if you look at the link to the terraform definition I put in a previous comment